README.md

```diff
@@ -1,105 +1,37 @@
-- Run containers, and build, push, and pull images (powered by [nerdctl])
-- Expose an application in Kubernetes for local access
+## Setup
 
-All of this is wrapped in an open-source application.
-
-[nerdctl]: https://github.com/containerd/nerdctl
-
-## Get The App
-
-You can download the application for macOS, Windows and Linux on the [releases page].
-
-[releases page]: https://github.com/rancher-sandbox/rancher-desktop/releases
-
-Running on Windows requires [Windows Subsystem for Linux (WSL2)]. This will be
-installed automatically during Rancher Desktop installation.
-
-[Windows Subsystem for Linux (WSL2)]:
-https://docs.microsoft.com/en-us/windows/wsl/install-win10
-
-Note, [development builds] are available from the CI system. Development builds
-are not signed.
-
-[development builds]:
-https://github.com/rancher-sandbox/rancher-desktop/actions/workflows/package.yaml?query=branch%3Amain
-
-For Linux, you will find both .DEB and .RPM packages attached as an asset for the most recent version.
-
-
-## System Requirements
-Recommended
-| **OS** | **Memory** | **CPU** |
-| :--- | :---: | :---: |
-| macOS | 8GB | 4CPU |
-| Windows | 8GB | 4CPU |
-| Linux | 8GB | 4CPU |
-
-OS versions we're currently using for development/testing:
-| **OS** | **Version**
-| :--- | :---: |
-| macOS | Catalina 10.15 or higher |
-| Windows | Home build 1909  or higher |
-| Ubuntu | Ubuntu 20.04 or higher |
-| openSUSE | Leap 15.3 or higher |
-| Fedora | Fedora 33 or higher |
-
-**Note:**
-Feel free to use a different OS version that haven't been listed.
-We are currently developing/testing Rancher Desktop on the listed OS and you can use it as a reference.
-
-## Base Design Details
-
-Rancher Desktop is an Electron application with the primary business logic
-written in TypeScript and JavaScript.  It leverages several other pieces of
-technology to provide the platform elements which include k3s, kubectl, nerdctl
-WSL, qemu, and more. The application wraps numerous pieces of technology to
-provide one cohesive application.
-
-## Building The Source
-
-Rancher can be built from source on macOS, Windows or Linux.  Cross-compilation is
-currently not supported.  The following provides some detail on building.
-
-### Prerequisites
-
-Rancher Desktop is an [Electron] and [Node.js] application. Node.js v16 is 
-recommended to build the source.  On Windows, [Go] is also required. On Linux,
-[QEMU] is required at runtime.
-
-[Electron]: https://www.electronjs.org/
-[Node.js]: https://nodejs.org/
-[Go]: https://golang.org/
-[QEMU]: https://www.qemu.org/
-
-#### Windows
+### Windows
 
 There are two options for building from source on Windows: with a
 [Development VM Setup](#development-vm-setup) or
 [Manual Development Environment Setup](#manual-development-environment-setup)
 with an existing Windows installation.
-##### Development VM Setup
+
+
+#### Development VM Setup
 
 1. Download a Microsoft Windows 10 [development virtual machine].
```

Comment:
We probably should have said something about doing the steps below in the VM… :D

Reply:
Ok I added a note about that in point 1. Let me know if you see any issues.